pageserve/templates/index.html

```diff
@@ -33,9 +53,13 @@ <h2>Past posts</h2>
 <p>{{post.text}}</p>
 {% endif %}
 {% if post.files %}
-  {% for file in post.files %}
-  <img src="{{ file }}">
-  {% endfor %}
+{% for file in post.files %}
+<img src="{{ file }}">
+{% endfor %}
+{% endif %}
+
+{% if post.author_id == session["user_id"] %}
+<a href='#' onClick="{{'deletePost(\'' + post._id['$oid'] + '\');'}}">Delete</a>
```

Comment:
This is a little hard to read, I think it's cleaner to just put `post._id['$oid']` in the braces:

```
onClick="deletePost('{{ post._id['$oid'] }}')"
```

Reply:
Thanks, much cleaner. Done https://github.com/knative-portability/large-events/pull/56/commits/e669f9dfbadb277761d5b45d24fe90e09c4acced